### ISSUE TITLE
allowTaggedTemplates isn't allowed on the config object

### ISSUE THREAD

**Issue by jonathan-potter** [ihordiachenko/eslint-plugin-chai-friendly#4]
It looks like the only place in this repo that `allowTaggedTemplates` is referenced is the readme. When `allowTaggedTemplates` is added to the configuration object eslint breaks with the message below. This isn't the end of the world, you just need to remove it from the readme or state that it does not work.

```sh
Error: [object Object]:
	Configuration for rule "chai-friendly/no-unused-expressions" is invalid:
	Value "[object Object]" should NOT have additional properties.
```

**Comment by ihordiachenko** [comment]
Thanks for your feedback! Looks like I just need to fetch the latest changes from the original `no-unused-expressions` rule. Will be added soon.

**Comment by ihordiachenko** [comment]
Will be included in the next release, alongside the #3
